Plantix/buyer-app/flow-2/confirm.json

```diff
@@ -0,0 +1,173 @@
+        "bap_uri": "https://newest-plants-phillips-probably.trycloudflare.com/ondc/v1",
+        "bpp_id": "preprod-ondc.addble.com",
+        "bpp_uri": "https://preprod-ondc.addble.com/",
+        "city": "std:080",
+        "core_version": "1.2.0",
+        "country": "IND",
+        "domain": "ONDC:AGR10",
+        "message_id": "4e93f5bf-c2b8-465f-85a0-42bd704b731a",
+        "transaction_id": "1c6bbef3-3577-477f-a77b-2176e998c645",
+        "timestamp": "2024-11-07T08:07:00.332471+00:00",
+        "ttl": "PT30S"
+    },
+    "message": {
+        "order": {
+            "id": "c0bdf507-e1c8-4a9e-8c8c-eaa001f2bf3d",
+            "state": "Created",
+            "billing": {
+                "address": {
+                    "name": "John's Home",
+                    "building": "Apt 101",
+                    "locality": "Main St",
+                    "city": "Sample City",
+                    "state": "Sample State",
+                    "country": "IND",
+                    "area_code": "123456"
+                },
+                "phone": "7702863154",
+                "name": "John Doe",
+                "email": "john.doe@example.com",
+                "created_at": "2024-11-07T08:04:47.682870+00:00",
+                "updated_at": "2024-11-07T08:04:47.682993+00:00"
+            },
+            "items": [
+                {
+                    "id": "F9nPDq0U58cOTPbOaWL1",
+                    "quantity": {
+                        "count": 2
+                    },
+                    "fulfillment_id": "43d79997-ac7d-4e0c-9b44-deca0e1a2da5"
+                },
+                {
+                    "id": "YZ2SySvvAuPhIyec6gog",
+                    "quantity": {
+                        "count": 3
+                    },
+                    "fulfillment_id": "43d79997-ac7d-4e0c-9b44-deca0e1a2da5"
+                },
+                {
+                    "id": "4vkgqsSvRZEwywbpMeW2",
+                    "quantity": {
+                        "count": 4
+                    },
+                    "fulfillment_id": "43d79997-ac7d-4e0c-9b44-deca0e1a2da5"
+                }
+            ],
+            "provider": {
+                "id": "H6JplBH0HQxkVoKWAnD3",
+                "locations": [
+                    {
+                        "id": "7kZu3N7P57bFinuL1yha"
+                    }
+                ]
+            },
+            "fulfillments": [],
+            "payment": {
+                "uri": "",
+                "tl_method": "",
+                "params": {
+                    "amount": "2150.00",
+                    "currency": "INR",
+                    "transaction_id": ""
+                },
+                "status": "NOT-PAID",
+                "type": "ON-ORDER",
+                "collected_by": "BAP",
+                "buyer_app_finder_fee_type": null,
+                "buyer_app_finder_fee_amount": null,
+                "settlement_details": null
+            },
```

Comment:
I think if `"type": "ON-ORDER"` what we also want we need to have `"status": "PAID"` here.
And of course finders fee is missing here.